From andre@bitwigglers.org  Mon Jan 26 09:41:59 2009
From: Andre Haupt <andre@bitwigglers.org>
Date: Mon, 26 Jan 2009 16:12:17 +0100
Subject: Staging: me4000: use linux/uaccess.h and linux/io.h
To: greg@kroah.com
Cc: Andre Haupt <andre@bitwigglers.org>, g.gebhardt@meilhaus.de, w.beiter@aon.at
Message-ID: <1232982745-16220-2-git-send-email-andre@bitwigglers.org>


This fixes the following checkpatch.pl warnings:

WARNING: Use #include <linux/uaccess.h> instead of <asm/uaccess.h>
WARNING: Use #include <linux/io.h> instead of <asm/io.h>

Signed-off-by: Andre Haupt <andre@bitwigglers.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/me4000/me4000.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/staging/me4000/me4000.c
+++ b/drivers/staging/me4000/me4000.c
@@ -36,8 +36,8 @@
 #include <linux/vmalloc.h>
 #include <linux/slab.h>
 #include <asm/pgtable.h>
-#include <asm/uaccess.h>
-#include <asm/io.h>
+#include <linux/uaccess.h>
+#include <linux/io.h>
 #include <asm/system.h>
 
 /* Include-File for the Meilhaus ME-4000 I/O board */
